src/main/scala/temple/generate/kube/KubernetesGenerator.scala

```diff
@@ -85,7 +122,7 @@ object KubernetesGenerator {
 
     mkCode(
       generateHeader(service, GenType.Deployment, isDb = true),
-      printer.pretty(deploymentBody),
+      this.printer.pretty(deploymentBody),
```

Comment:
Is there a reason why we’re using `this` everywhere now? Is there another printer in scope locally?

Reply:
I just noticed I'd used both in different places and chose one for consistency, happy to change if needed